eksctl/nasa-cryo.jsonnet

```diff
@@ -0,0 +1,109 @@
+// Exports an eksctl config file for carbonplan cluster
+local ng = import "./libsonnet/nodegroup.jsonnet";
+
+// place all cluster nodes here
+local clusterRegion = "us-west-2";
+local masterAzs = ["us-west-2a", "us-west-2b", "us-west-2c"];
+local nodeAz = "us-west-2a";
+
+// Node definitions for notebook nodes. Config here is merged
+// with our notebook node definition.
+// A `node.kubernetes.io/instance-type label is added, so pods
+// can request a particular kind of node with a nodeSelector
+local notebookNodes = [
+    { instanceType: "m5.large" },
+    { instanceType: "m5.xlarge" },
+    { instanceType: "m5.2xlarge" },
+    { instanceType: "m5.8xlarge" },
+];
+
+// Node definitions for dask worker nodes. Config here is merged
+// with our dask worker node definition, which uses spot instances.
+// A `node.kubernetes.io/instance-type label is set to the name of the
+// *first* item in instanceDistribution.instanceTypes, to match
+// what we do with notebook nodes. Pods can request a particular
+// kind of node with a nodeSelector
+local daskNodes = [
+    { instancesDistribution+: { instanceTypes: ["m5.large"] }},
+    { instancesDistribution+: { instanceTypes: ["m5.xlarge"] }},
+    { instancesDistribution+: { instanceTypes: ["m5.2xlarge"] }},
+    { instancesDistribution+: { instanceTypes: ["m5.8xlarge"] }},
+];
+
+{
+    apiVersion: 'eksctl.io/v1alpha5',
+    kind: 'ClusterConfig',
+    metadata+: {
+        name: "nasa-cryo",
+        region: clusterRegion,
+        version: '1.22'
+    },
+    availabilityZones: masterAzs,
+    iam: {
+        withOIDC: true,
```

Comment:
@sgibson91, I might be far from truth, but I noticed https://github.com/2i2c-org/infrastructure/blob/bf6ae45b97d27cf303c68379c35adf9d493097cf/eksctl/uwhackweeks.jsonnet#L62-L66 that is missing here.

Shouldn't we add the `nasa-cryo-staging` and the prod one here and then list the correspoding service account in singleuser, similar to https://github.com/2i2c-org/infrastructure/pull/1768/commits/559c1e384635c75d55c791380d40fb93a0160dc1 ?

Reply:
I can still see the `cloud-user-sa` in the logs `serviceaccount \"cloud-user-sa\"` for some reason
````

Reply:
I removed it from the hub config, not here. Am I supposed to?

Reply:
I've never used the jsonnet files before so I am clueless and completely dependent on @yuvipanda's `generate-cluster` command to "do the right thing" here

Reply:
>  logs serviceaccount \"cloud-user-sa\" for some reason

Did you check the prod logs? I probably only ran the update against staging to debug

Reply:
I updated both hubs just now

Reply:
Yes, I was in the prod hub sorry for the cofusion :(

Reply:
I added the prod/staging annotations in their respective `<hub>.values.yaml` files, like:

https://github.com/2i2c-org/infrastructure/pull/1768/files#diff-5204ecf6a357bcb3d7cff3005b283cf99acd6af7b4ff054e6ab13df64a198e56R2-R4

I have no idea if they are the same/different to `cloud-user-sa`